src/components/SUSHIControls.js

```diff
@@ -176,6 +199,44 @@ export default function SUSHIControls(props) {
     props.onClick(true, jsonOutput, isObject);
   }
 
+  function ExampleGrid() {
+    let columns = [];
+    let components = [];
+    let obj = Object.entries(config);
+    for (let group = 0; group < obj.length; group++) {
+      components.push([]);
+      for (let file = 0; file < obj[group][1].files.length; file++) {
+        let fileName = obj[group][1].files[file].name;
+        let fileLink = obj[group][1].files[file].link;
+        components[group].push(
+          <Grid item xs>
```

Comment:
I'm getting a warning in the console that says `Warning: Each child in a list should have a unique "key" prop.`. I think that is because these `Grid` items are coming from an array. There's a bit of documentation about it [here](https://reactjs.org/docs/lists-and-keys.html#keys). I think we can add a "key" prop and it should be happier.